@ndlib/gatsby-theme-marble/src/components/Pages/MarbleItem/ItemLayout/sx.js

```diff
@@ -0,0 +1,48 @@
+module.exports = {
+  mainMetadata: {
+    '& > dl > div': {
+      backgroundColor: 'transparent !important',
+      borderBottom: '1px solid',
+      borderColor: 'gray.1',
+      position: 'relative',
+    },
+    '& > dl > div:first-of-type': {
+      borderTop: '1px solid',
+      borderColor: 'gray.1',
+    },
+    '& dt': {
+      position: 'absolute',
+      verticalAlign: 'top',
+      width: ['120px', '200px', '200px'],
+    },
+    '& dd': {
+      display: 'inline-block',
+      marginLeft: ['120px', '200px', '200px'],
+      verticalAlign: 'top',
+      width: ['calc(100% - 120px)', 'calc(100% - 200px)', 'calc(100% - 200px)'],
+    },
```

Comment:
This minus the '& >' and '&' might be able to be moved the the main theme or global css.

Reply:
I decided against this.  Since it seems a bit like a site-specific choice and one that would only apply to the metadata list.